server/protocols/jmap/src/main/java/org/apache/james/jmap/model/OldKeyword.java

```diff
@@ -30,20 +30,24 @@
     private final Optional<Boolean> isFlagged;
     private final Optional<Boolean> isAnswered;
     private final Optional<Boolean> isDraft;
+    private final Optional<Boolean> isForwarded;
 
     @VisibleForTesting
-    OldKeyword(boolean isUnread, boolean isFlagged, boolean isAnswered, boolean isDraft) {
```

Comment:
We may think of introducing a builder for this class.

Reply:
not needed IMHO: we won't introduce other keyworkds, and this class is meant to be removed (soon I hope)